src/test/functional/cypress/support/oxd.js

```diff
@@ -45,6 +45,7 @@ const OXD_ELEMENTS = {
   inputGroup: '.oxd-input-group',
   pageContext: '.oxd-layout-context',
   pageTitle: '.orangehrm-main-title',
+  numRecords: '.oxd-text--span',
```

Comment:
@Super-Chama is this selector enough?

Reply:
seems not, i'll have to check this one

Reply:
there are no classes or identifier for this. I will keep this noted. I may have to make a new PR and refactor these with more custom methods

Reply:
there are no classes or identifier for this. I will keep this noted. I may have to make a new PR and refactor these with more custom methods